src/Gurucomkz/Watermark/Shortcode.php

```diff
@@ -0,0 +1,28 @@
+<?php
+namespace Gurucomkz\Watermark;
+
+use PageController;
+use SilverStripe\Control\Controller;
+use SilverStripe\Assets\Image;
+
+class Shortcode
+{
+
+    public static function Watermark($arguments, $content = null)
+    {
+
+        if (!isset($arguments['file'])) {
+            return '';
+        }
+
+        $image = Image::get()->filter("Name", $arguments['file'])->first();
```

Comment:
File names can change. I advise using IDs. 
Additionally, in order to help people writing these, you could add a field on the image viewer where the shortcode string is given for copy-paste.
i.e. somewhere here - a field with value `[shortcode id=123]`
<img width="441" alt="Screen Shot 2021-02-24 at 12 17 03" src="https://user-images.githubusercontent.com/12028675/108920857-49caca00-769a-11eb-9edc-e3961c3a6e7e.png">